Keas.Mvc/Controllers/ConfirmController.cs

```diff
@@ -177,19 +178,27 @@ public async Task<IActionResult> AcceptAll()
             var people = await _context.People.Where(p => p.User == user).Select(a => a.Team).AsNoTracking().ToArrayAsync();
             var teamAdmins = await _context.TeamPermissions.Where(tp => tp.User == user).Select(a => a.Team).AsNoTracking().ToArrayAsync();
             var teams = people.Union(teamAdmins, new TeamComparer());
-            if(teams.Count() == 0){
+
+            var groupPermissions = await _context.GroupPermissions.Include(a => a.Group).ThenInclude(a => a.Teams).ThenInclude(a => a.Team).AsNoTracking().Where(a => a.UserId == user.Id).ToListAsync();

```

Comment:
Why do we need to include teams->team?  Shouldn't we just care which groups they have access to and ignore teams that are part of those groups?

Reply:
@srkirkland Good point. Was doing it a different way at first. I'll fix tomorrow.